### ISSUE TITLE
Make astrokat build publicly available

### ISSUE THREAD

**Issue by sw00** [ska-sa/astrokat#69]
The automated build of `astrokat` currently happens on our private Jenkins instance. Since this is a public repository, the build information should be accessible publicly. 

To solve this, we can either:
- provide read-only access to the Jenkins build at a publicly available URL 
OR
- move the build to a platform like Travis or CircleCI which has a free tier for open source projects

I'm personally in favour of the latter since it's a common practice in the open source community.

**Comment by ajoubertza** [comment]
Agree, option 2 is the way to go.  Some of the build steps can't be moved though (e.g., publishing to our private pypi repo), so we'll probably end up with just those steps still running on the Jenkins pipeline, while test are done by Travis or CircleCI.
